chia/wallet/puzzles/p2_singleton.clvm

```diff
@@ -0,0 +1,61 @@
+(mod (SINGLETON_MOD_HASH SINGLETON_MOD_HASH_HASH SINGLETON_ID singleton_innerpuzhash my_id)
```

Comment:
First line should be comments about what this is all about (like "This puzzle is used to pay to an account that can be considered to be controlled by the inner puzzle of a singleton. The coins can only be moved in cooperation with the singleton also being spent simultaneously.")